pkg/base/models.go

```diff
@@ -37,6 +39,11 @@ type Tcpdump struct {
 	TcpOnly     bool   `long:"tcponly" description:"Only watch tcp connections"`
 }
 
+type UpgradeInspector struct {
```

Comment:
Please rebase `main` branch and move this configuration to `upgradeinspector` package.

Reply:
Done